### ISSUE TITLE
Game randomly crashes

### ISSUE THREAD

**Issue by TheBlulitron** [CreativeMD/LittleFrames#75]
Hello again,
the game started crashing again. This time however, it actually gave me a crash report:
https://pastebin.com/iWdLY0B8
I'm running Mod Version v.1.0.12 on Minecraft 1.12.2
VLC Version is 3.0.20 Vetinari

**Comment by CreativeMD** [comment]
Seems like something is going wrong there. Unfortunately I do not support 1.12 anymore. Maybe switch to the unofficial version? Cannot help you further there, sorry.

**Comment by TheBlulitron** [comment]
I did an oopsie on my end and somehow managed to install the new unofficial version on the server end, but not on the client end. I changed that now (now running LF v.1.1.4 and watermedia 2.0.28).
Now my game will immediately crash upon loading into the world when using Singleplayer:
https://pastebin.com/pPrnYkyG
On Multiplayer, I can login. However, everything from LittleTiles becomes invisible and the game crashes when opening the inventory:
https://pastebin.com/9rAgjPgu

I hope I'm not bothering you guys too much...

**Comment by CreativeMD** [comment]
These are very weird crashes. Almost seems like something went wrong when installing Forge.

**Comment by TheBlulitron** [comment]
I just tried reinstalling Forge with it's latest version of 1.12.2 (2860)
Still the same issue unfortunately...

**Comment by TheBlulitron** [comment]
I also found this in the log:
https://pastebin.com/3dLqv14w
I'm no expert, but could it be, that something collides between CreativeCore, LittleFrames and LittleTiles?

**Comment by CreativeMD** [comment]
Is that an error on server or client side? All this error says that a class cannot be loaded. It is a class from Forge. There is nothing I can do about that unfortunately.

**Comment by TheBlulitron** [comment]
The errors are all client-side. The server itself won't crash. It will just state that I disconnected due to an internal exception.
I can't wrap my head around it either. Forge is working fine except for this particular problem. Since I'm probably the only person with this issue, I can only assume it's a mod conflict?
Nevertheless I backported to the latest version without Watermedia. While it is still very unfortunate that most images just won't load properly (apparently it cannot parse certain Strings when the images are being loaded from imgur), at least my game won't crash anymore and my world is "saved"...
